test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/InstrumentationPassIntegrationTest/BasicTest.cshtml

```diff
@@ -3,4 +3,7 @@
 @("Hello")
 <form unbound="foo">
     <input value=Hello date=@DateTime.Now type='text' />
-</form>
\ No newline at end of file
+</form>
+
+<span>Here is some content @*with a comment*@</span>
```

Comment:
I don't see a ton of value to it, this is already a test specifically for the instrumentation feature.